standard_releases.md

```diff
@@ -1,59 +1,215 @@
+
+* Make a new branch from `master` for the translation work
+* Review, test, snag and finally merge this branch into `master`
+* Make a new branch from the current release and bring across the new
+  translations from Transifex.
+* As before, review this branch and fix any snagging issues with the translation
+  in this context. It is assumed that normally these fixes will not need to be
+  merged back into master, but if they uncover bugs not previously found, those
+  should be `git cherry-pick`-ed back into `master`.
+
+When `master` and the current release have diverged significantly (a situation
+we've yet to encounter), it's assumed that the correct process will be to treat
+the translation of each as a separate project. In this case, it's probably most
+helpful to translate the current release first, bring across any fixes and only
+initiate translation of `master` when it's due for release.
+
+### Fixes to an existing release
+
+When a bug is discovered or a correction needs to be made to an existing
+release and it is deemed severe enough to require an immediate fix (i.e.
+it cannot wait for the next naturally-occurring release), then we need
+to make an unplanned release. The process for this is as follows:
+
+* Make a new branch named something like `v0-2-0-description-of-issue`
+  from the current release branch and commit the fix(es) to it
+* Review and test this branch in a pull-request
+* If necessary, perform necessary translation work on this branch once this fix
+  is approved in English. Whether this is necessary depends on the nature of the
+  fix, i.e. whether it changes translated text.
+* At this point, we need to decide on the impact of the changes on users of the
+  standard and the appropriate new version number. Create a new release branch
+  named accordingly, e.g. v0-2-1 or v0-3-0 and merge the fixes into it.
+* Follow the usual release process of enabling, updating to latest, etc, so that
+  this release becomes the new 'current' release. If the issue warrants it,
+  consider adding a notice to the old release documenting the issue and that it
+  should not be used. You may also have to notify existing users of that
+  version.
+* If the fixed issue is also present in `master`, the fix branch should also be
+  merged back into there, in a separate pull-request.
+
+**Note:** until we hit a stable 1.0 release, only very severe bugs that impede real
+world use of the standard, present a security risk or otherwise cause significant
+damage to our aims as an organisation should require this kind of fix. It is also
+assumed that other older pre 1.0 releases are automatically and immediately
+considered end-of-life when a new version is released. This policy is likely to
+change only when we have real world use or commitments mandating a greater level
+of support.
+
+### Fixes to an existing translation
+
+Fixes that solely relate to an existing translation (i.e. not the source text,
```

Comment:
Fixes that solely relate to an existing translation (i.e. not the source text, but a particular translation of it) should have an issue raised against them in Github. The fix can then be made by editing the translation in Transifex and the updated strings pulled down to a local branch named for the issue eg `297-concept-translation` which should then be pushed up and merged directly back into the existing release branch.
This sort of fix does not require a change in version number or a new release branch

Reply:
I've made a change pretty similar to this in 72ac58a. Now that the section above mentions github issues, I've kept the reference to that, but otherwise added the rest.